src/agent.py

```
@@ -437,6 +437,19 @@ async def async_chat(
         "Got response", response_preview=response_text[:50], response_id=response_id
     )
 
+    # Guardian: fire-and-forget safety/faithfulness evaluation (never blocks response)
+    from config.settings import GUARDIAN_ENABLED
+    if GUARDIAN_ENABLED:
+        from services.guardrail_service import guardrail_service
+        asyncio.create_task(
```

Ruff (0.15.14)

[error] Undefined name `asyncio`

(F821)